vendor-extra/ViewsBundle/tests/Views/StringViewConverterTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace tests\Libero\ViewsBundle\Views;
+
+use Libero\ViewsBundle\Views\CallbackViewConverter;
+use Libero\ViewsBundle\Views\StringView;
+use Libero\ViewsBundle\Views\StringViewConverter;
+use Libero\ViewsBundle\Views\TemplateViewBuildingViewConverter;
+use Libero\ViewsBundle\Views\View;
+use Libero\ViewsBundle\Views\ViewConverter;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use tests\Libero\LiberoPageBundle\ViewConvertingTestCase;
+use tests\Libero\LiberoPageBundle\XmlTestCase;
+
+final class StringViewConverterTest extends TestCase
+{
+    use ViewConvertingTestCase;
+    use XmlTestCase;
+
+    /**
+     * @test
+     */
+    public function it_is_a_view_converter() : void
+    {
+        $converter = new StringViewConverter($this->createFailingConverter());
+
+        $this->assertInstanceOf(ViewConverter::class, $converter);
+    }
+
+    /**
+     * @test
+     * @dataProvider visibleNodeProvider
+     */
+    public function it_returns_a_string_view_for_visible_nodes(string $node, string $expected) : void
+    {
+        $converter = new StringViewConverter($this->createFailingConverter());
+
+        $this->assertEquals(
+            new StringView($expected, ['bar' => 'baz']),
+            $converter->convert($this->loadNode($node), 'template', ['bar' => 'baz'])
+        );
+    }
+
+    public function visibleNodeProvider() : iterable
+    {
+        yield 'cdata' => ['<![CDATA[<cdata>]]>', '<cdata>'];
```

Comment:
always confusing to work with `CDATA`, could the content be `42 > 41` rather than `<cdata>` so that it is generic character data rather than another "escaped" element which is a specific use case?